File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/Linkage.ts
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { BackboneElement } from "./BackboneElement";

export type LinkageItem = BackboneElement & {
    type: string;
    resource: Reference;
};

export type Linkage = DomainResource & {
    item: LinkageItem[];
    active?: boolean;
    author?: Reference;
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/Linkage.py
from typing import Optional, List
from base import DomainResource
from base import Reference
from base import BackboneElement

class Linkage_Item(BackboneElement):
    type: str
    resource: Reference

class Linkage(DomainResource):
    item: list[Linkage_Item] = []
    active: Optional[bool] = None
    author: Optional[Reference] = None